gitconfig

```diff
@@ -58,13 +58,18 @@
 	autosetuprebase = always
 
 [diff]
-	tool = bc3
-	# optionally, set `external = difft`
+	# tool = bc
+	tool = difftastic
```

Comment:
Removing `external = difft` fixed the fugitive output... I suspect messing with `git diff` default output is not preferred.